pihole

```diff
@@ -509,9 +504,8 @@ Options:
   enable              Enable Pi-hole subsystems
   disable             Disable Pi-hole subsystems
                         Add '-h' for more info on disable usage
-  restartdns          Full restart Pi-hole subsystems
-                        Add 'reload' to update the lists and flush the cache without restarting the DNS server
-                        Add 'reload-lists' to only update the lists WITHOUT flushing the cache or restarting the DNS server
+  reloaddns           Update the lists and flush the cache without restarting the DNS server
+  reloadlists     Update the lists WITHOUT flushing the cache or restarting the DNS server
```

Comment:
Just using the same indentation for every item:

```suggestion
  reloadlists         Update the lists WITHOUT flushing the cache or restarting the DNS server
```

Reply:
I've aligned them all in one fell swoop!